ansible/idr-elasticsearch.yml

```diff
@@ -133,8 +133,9 @@
           - "{{ apps_folder }}/searchengine/elasticsearch/certs: /certs"
           - "{{ apps_folder }}/searchengine/elasticsearch/certs:/usr/share/elasticsearch/config/certificates"
 
-
-
+  - name: Pause for 1 minutes
+    ansible.builtin.pause:
+      minutes: 1
```

Comment:
Rather than pausing for an arbitrary amount of time. Would a different `state` e.g. `stopped` in the previous task allow to ensure that the command is completed successfully before executing the rest of the playbook - https://docs.ansible.com/ansible/2.9/modules/docker_container_module.html#parameters?

Reply:
I have used `wait_for` to ensure the ca file is present instead of `pause` for an arbitrary amount of time.